.gitignore

```diff
@@ -15,3 +15,5 @@ stacktrace*
 test/.coverage-unit
 tmp
 sftp-config.json
+*.swp
```

Comment:
my editor generates while editing, I don't think we'd be needing these file types. So, I think it's safe to ignore them.